ompi/mpi/c/info_f2c.c

```diff
@@ -27,6 +27,8 @@
 #include "ompi/mpi/c/bindings.h"
 #include "ompi/runtime/params.h"
 #include "ompi/errhandler/errhandler.h"
+#include "ompi/communicator/communicator.h"
+#include "ompi/errhandler/errhandler.h"
```

Comment:
Why add these includes?

Reply:
add more lines of source code I guess.  part of an iteration process.  i'll back these out.